Gemfile

```diff
@@ -21,25 +21,23 @@ gem 'net-ldap'
 
 # Use puma as the webserver in development
 gem 'puma'
-gem 'rack', '1.6.4' # Remove in rails 5
```

Comment:
I haven't noticed any docker issues when running rspec, but that doesn't kick off a puma server though.